### DISCUSSION TITLE
一个河北的购房者是这样的处境

### DISCUSSION THREAD

**Discussion by 1510207073** [WeNeedHome/SummaryOfLoanSuspension#556] [General]
买房时：

1. 交6w返8w，实际上这个是电商费；
2. 买房送车位，实际上是车位使用权，每个月都要交管理费；
3. 有北师大幼儿园，实际上是冒用的北师大名字，后来被起诉赔了北师大 80w;
4. 房屋实际销售价格高于房屋备案价很多；
5. 房屋建造是中建二局，实际上后来换了好几波承包商；
6. 违规首付分期；

走法律途径维权：
后来部分业主走法律途径将上述的违法违规行为上诉到当地的法院，后判决败诉，律师讲开完庭后法官就直接说这种事情不会让你们胜诉的，据说上面有文件。

后来业务自发的去市政府维权：
市政府统一督促开发商出一个详细的施工进度表，并督促开发商按照计划执行，结果到期并没有完工。中间尝试打过各种市长省长热线留言，没有任何卵用。前段时间又去市政府维权，业主要求政府出面督促开发商有更实质进展，被抓了十几个，要求业主退出维权群。

**Comment by zxcs001** [comment]
国内卖车位，就是纯粹的割韭菜。

这东西压根没有产权的，所有车位的产权都是人防的。也就是说，你买的使用权，真的闹到打官司的时候，国家是可以不保护的。

承包商没啥区别，中建二局接了也是要分包的，别以为国字头有什么区别，都是草台班子。

首付分期是因为老百姓实在没钱了，吸白粉的就是这么一回事，谁管得了未来啊。这要爆炸了，比次级贷还要厉害，首付贷，贷上贷。

**Comment by kennard520** [comment]
强制停供

**Reply by a6749920** [reply]
全家贷开启了，18贷，贷贷相传，亲戚贷，朋友贷，不远了，我想到了这些新词哈哈

**Reply by tencentbook** [reply]
灵活停供

**Reply by ruo-xi** [reply]
恶意停供

**Comment by kydyahah** [comment]
你说的是涿州铂悦山吗？

**Reply by 1510207073** [reply]
是

**Reply by kydyahah** [reply]
我也是

**Reply by 1510207073** [reply]
法治法治法治爱国和谐诚信富强自由和谐和谐民主和谐平等和谐民主和谐富强和谐文明和谐富强和谐法治和谐富强和谐法治和谐和谐

**Comment by ldmid666** [comment]
河北廊坊，泰禾小区，从20年11月到22年，延期将近一年半，起诉退房后，全部诉求被驳回，最后才算延期49天，被法院气死，没有公平可言。
赠送车位，最后全是很小的机械车位

**Reply by 1510207073** [reply]
环京都一个吊样

**Reply by WallStreetHu** [reply]
泰禾要破产的节奏

**Comment by SystemCodes** [comment]
![FC829156-C6A8-4AA0-9568-309A2E48F7FF](https://user-images.githubusercontent.com/109196939/179184952-3e981f5e-ca5d-4251-b10b-ae0042afa679.jpeg)
你知道图中这位江西老表前后在说什么吗？
大概意思是上面口头说就行了，不需要发什么文件的，权力大于法律，所以你说什么法律不是很好笑吗？

法律是上位者针对下位者的，假如你是派出所所长，平头老百姓犯法你抓就抓了，但又有条例保护全国人大和官员，哪怕犯法小小的所长是不能抓他们的，除非得到更高授权。

开发商与地方员勾结，法院根本就不会受理什么开发商诈骗，诈骗又如何，因为你是下位者，骗就骗了。

疫情时候法院和律师收到指示不准受理疫情相关的案件，你被打被骂，搞烂你家东西了，杀你家宠物了，你就得自认倒霉。

市区法院也是受市领导班子指示的，一般都是市委副级任公安和法院一把手。
地方要开发商买地皮，维持高房价，这样能刷GDP数据，地皮的钱已经到手，银行也把钱贷出去坐收利息了，至于房子烂不烂尾，购房者有没有房子住无所谓的。

本来就是政府、银行、开发商一起压榨老百姓，最后房子很差甚至烂尾，这和他们有什么关系呢。

很多事情不可能通过法院解决的，停贷或把事情闹大，闹到老外都知道，让最上位者的脸色不好看，最上位者就会给上位者施压，然后问题自然解决了。
